src/content/tutorials/fleet-management/cluster-management/cluster-autoscaler/index.md

```diff
@@ -0,0 +1,125 @@
+---
+linkTitle: Cluster autoscaler
+title: Advanced cluster autoscaler configuration
+description: Here we describe how you can customize the configuration of the managed cluster autoscaler service in your workload clusters.
+weight: 90
+menu:
+  principal:
+    parent: tutorials-fleet-management-clusters
+    identifier: tutorials-fleet-management-clusters-cluster-autoscaler
+user_questions:
+  - Where can I find the ConfigMap to configure cluster-autoscaler?
+  - What cluster-autoscaler options can I configure?
+last_review_date: 2024-11-29
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+Your Giant Swarm installation comes with a default configuration for the [cluster-autoscaler addon](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler). You need access to the platform API to configure the cluster-autoscaler. [Learn how to access the platform API]({{< relref "/getting-started/access-to-platform-api" >}}).
```

Comment:
Currently, the `cluster-autoscaler` is running on the WC, so there is one per WC, instead of one `cluster-autoscaler` per installation. Also, not all providers currently support `cluster-autoscaler`, i.e CAPZ does not support `cluster-autoscaler`.